help/destinations/catalog/crm/salesforce.md

```diff
@@ -57,14 +61,14 @@ Create the custom field of type `Text Area Long` which Experience Platform will
 Refer to the Salesforce documentation to [create custom fields](https://help.salesforce.com/s/articleView?id=sf.adding_fields.htm&type=5) if you need additional guidance. 
```

Comment:
minor, applies to line 60 - missing comma:

Create a custom field of type `Text Area Long`, which Experience Platform will use